ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/activityBlock/activityBlock.component.html

```diff
@@ -13,6 +13,16 @@
                 </ddp-modal-activity-block>
             </div>
         </ng-container>
+        <ng-template #embedded>
+          <div *ngFor="let instance of childInstances; trackBy: getInstanceId" class="card">
```

Comment:
You have this line twice. I suggest simplifying it and using 
`*ngIf="isModal; else embedded"` conditional inside of the *ngFor loop